src/styles.scss

```diff
@@ -1287,3 +1279,8 @@ mat-tab-group.no-pagination {
     flex-direction: column;
   }
 }
+
+.custom_copy_snack_bar {
+  min-width: 0 !important;
```

Comment:
Old-school CSS rule-of-thumb is to avoid `!important`, if possible. May not apply as much any more, but do you need it?

Reply:
Unfortunately I believe so, since snack bar already has their own styling for `min-width`, in order to override it, I had to apply `!important`